.github/workflows/basic-tests.yml

```diff
@@ -35,6 +35,7 @@ jobs:
         git lfs pull -I "data/lalsimulation/SEOBNRv4ROM_v2.0.hdf5"
         mv data/lalsimulation/* ../
         cd ../
+        curl https://git.ligo.org/waveforms/software/lalsuite-waveform-data/-/raw/main/waveform_data/SEOBNRv4ROM_v3.0.hdf5 > SEOBNRv4ROM_v3.0.hdf5
```

Comment:
```suggestion
        curl --show-error --silent --remote-name https://git.ligo.org/waveforms/software/lalsuite-waveform-data/-/raw/main/waveform_data/SEOBNRv4ROM_v3.0.hdf5
```

Reply:
In fact, we may as well just use curl for the files above and get away with all the git/cd/mv commands.

Reply:
SEOBNR_v2_ROM is like 20 separate files, which might be messy. But let me see if just getting the v4 files is sufficient now.

Reply:
Drop the `>` redirects, that is the point of `--remote-name`.